pkg/utils/namespaceOffloading_conditions_update.go

```diff
@@ -0,0 +1,48 @@
+package utils
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	offv1alpha1 "github.com/liqotech/liqo/apis/offloading/v1alpha1"
+)
+
+// UpdateNamespaceOffloadingCondition -> if present update condition of that type, otherwise create a new one.
```

Comment:
```suggestion
// UpdateNamespaceOffloadingCondition updates the content of the given condition for the selected Cluster ID.
```

I personally feel it more standard. You could do the same also for the other one

Reply:
thank you